README.md

```diff
@@ -6,3 +6,6 @@ My first repo of microverse
 Michael Hirpo
 mikaelhirpo@gmail.com
 
+## Company
+Microverse INC
+
```

Comment:
- [ ] Kindly be noticed that the README file should be descriptive? AKA: Microverse README template should be used.

Reply:
what is the template?

Reply:
This is the [README Template](https://github.com/microverseinc/readme-template/blob/master/README.md).

Reply:
Thank you @ShahierNashaat  I have made the changes now and i have commited.